pkg/azuredisk/azuredisk_v2.go

```diff
@@ -368,19 +371,28 @@ func (d *DriverV2) StartControllersAndDieOnExit(ctx context.Context) {
 	// Leader controller manager should recover CRI if possible and clean them up before exiting.
 	go func() {
 		<-mgr.Elected()
+		var errors []error
+		ctx, w := workflow.New(ctx)
+		defer func() {
+			if len(errors) == 0 {
+				return
+			}
+			err := status.Errorf(codes.Internal, "%v+", errors)
```

Comment:
You've added support to `Finish` to handle multiple errors, so this is no longer necessary.